proj/funcDictionary.py

```diff
@@ -0,0 +1,335 @@
+# info: some general functions:
+
+import numpy as np
+import networkx as nx
+
+#import pyspark
+from imnet import process_strings as p  # used?
+
+#import subprocess
+#import re
+#from IPython.display import HTML
+
+#import sparkhpc
+#from sparkhpc import sparkjob
+#from sparkhpc.sparkjob import LSFSparkJob
+#import findspark
+#from scipy.sparse import csr_matrix
+
+#from plotData import PlotData
+#from plotData import InnerDataLevenshtein
+#from plotData import PlotDataLevenshtein
+
+def make_nets(ls, seq): # We still need seq? What is that?
+    """
+    info: make a list "nets", where each element "net" is 
+        a list again, containing all sequences with a certain 
+        length
+    input:
+        ls:  list of length values for each sequence in seq
+        seq: array of nucleic acid sequences 
+    output: 
+        nets: list of nets, where each element "net" is 
+        a list again, containing all sequences with a certain 
+        length  
+    """
+
+    lsHist = np.histogram(ls, bins = np.arange(0, 120, 3))
```

Comment:
Shouldn't the binning size be a variable and not just simply set to 3?

Reply:
"ls" is an array of over all lengths. In a nucleic acid strain, 3 nucleic acids always encode 1 amino acid. Thus the length is always a multiple of 3. That's why the binning size is 3. Otherwise, every value, 
which is not a multiple of 3, would be zero.  This value will always be 3 and is not changed.